server/bot.go

```diff
@@ -16,16 +17,17 @@ func (p *Plugin) PostBotDM(userID string, message string) {
 }
 
 // PostBotCustomDM posts a DM as the cloud bot user using custom post with action buttons.
-func (p *Plugin) PostBotCustomDM(userID string, message string, todo string, issueID string) {
+func (p *Plugin) PostBotCustomDM(userID, message, todo, postPermalink, issueID string) {
 	p.createBotPostDM(&model.Post{
 		UserId:  p.BotUserID,
 		Message: message + ": " + todo,
 		Type:    "custom_todo",
 		Props: map[string]interface{}{
-			"type":    "custom_todo",
-			"message": message,
-			"todo":    todo,
-			"issueId": issueID,
+			"type":          "custom_todo",
+			"message":       message,
+			"todo":          todo,
+			"postPermalink": postPermalink,
```

Comment:
What happens for posts that do not have this prop?

Reply:
If we don't have this prop, we won't have a permalink in the message in the `todo` sent to other user in the post